.env.development

```diff
@@ -1 +1,2 @@
 NEXT_PUBLIC_CFM_API_URL=http://127.0.0.1:3000
+AWS_API_URL=https://api-dev.communityfridgefinder.com/v1/
```

Comment:
This is the prod API url by the way: https://api-prod.communityfridgefinder.com/v1/